lib/spells/effects/Clone.cpp

```diff
@@ -28,9 +28,7 @@ namespace effects
 
 VCMI_REGISTER_SPELL_EFFECT(Clone, EFFECT_NAME);
 
-Clone::Clone()
-	: UnitEffect(),
-	maxTier(0)
+Clone::Clone() : maxTier(0)
```

Comment:
why did clang-tidy decide not to initialise `maxTier` in declaration? :)

Reply:
I do not know. I ran 2 versions of it, and this is actually version 14. Maybe in version 15 it have better algorithms.

Reply:
Done by hand